docs/tutorials/pre_executed/zubercal-ps1-snad.ipynb

```diff
@@ -0,0 +1,2344 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    snad_catalog = from_dataframe(
</pre><p>This might be unnecessary with Olivia's latest changes to crossmatch from a dataframe. Though it is nice to show the in-memory dataframe.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/444/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I think I do it other way around, snad is the "left" catalog in my pipeline</p>